twilight-http/src/request/application/emoji/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod add_emoji;
+pub mod delete_emoji;
+pub mod list_emojis;
+pub mod update_emoji;
```

Comment:
import and usage wise it might be better if the modules are private, and the structs are re-exported. this is also how the other modules are setup already